quickstarts/newrelic-ingest-metric/config.yml

```diff
@@ -0,0 +1,51 @@
+# Name of the quickstart (required)
+name: newrelic-ingest-metric
+
+# Displayed in the UI (required)
+title: New Relic Ingest Metric
+
+# Long-form description of the quickstart (required)
+description: |
+  This quickstart provides a synthetic monitor (and associated dashboard) that will regularly query ingest for each data type (APM Events, Infrastructure Processes, etc.) and post it to the Metric API as a custom metric `newrelic-ingest-metric`. This means that you can query and analyze ingest across your account for 13 months by default, instead of 8 days for events, 30 days for  Logs, etc. The custom metric also includes a facet for each data type, so that you can see, for example, ingest for Mobile Events by appName or Logging by labels.app. The following parameters have default values but are all configurable:
+  - ingest interval (hourly, daily, weekly)
+  - facet for each data type
+  - name of custom metric
+  - addition of custom events
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Use this script to set up a synthetic monitor that:
+  1. Queries ingestion by data source, faceted by the attribute of your choice (e.g., APM Events by appName)
+  2. Transforms the output into a metric that is pushed to the Metric API
+  3. Can be queried for consumption analysis / dashboards for 13 months
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Nora Shannon Johnson
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - ingest
+  - ingestion
+  - custom metric
+  - synthetic monitor
+  - consumption
+  - billing
+  - facet
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - infra-agent-standard
+
+documentation:
+  - name: nr1-ingest-metric-script (github)
+    url: https://github.com/umanorapracadadia/nr1-ingest-metric-script
```

Comment:
After considering this section I have some concerns that we are pointing users to a personal github account that is falling outside the new relic organization so there is no org level over-sight or security monitoring on this personal repo. So I'll find some time to talk more about what you are attempting to accomplish here.

Reply:
we've recommended moving this code into a new experimental repo in the new relic org, and shared that process in slack. Let us know if you have questions. Thanks!